RoboJackets/robocup-software
# Seeking Behavior

## Description
Adding seek behavior to offense.

## Associated / Resolved Issue
Resolves [ClickUp](https://app.clickup.com/t/389zmk)

## Design Documents
[Link](link-to-design-doc)

## Steps to Test
make run-sim (forced offense to always search)

**Expected result:** Travels to open position

## Key Files to Review
 * offense.cpp
 * offense.hpp
 * seeking.cpp
 * seeking.hpp

## Review Checklist


## (Optional) Sub-issues (for drafts)
_Note: if you find yourself breaking this PR into many smaller features, it may make sense to break up the PR into logical units based on these features._
